content/shmem_team_destroy.tex

```diff
@@ -1,37 +1,39 @@
 \apisummary{
-    shmem\_team\_destroy function destroys existing team.
+    Destroys existing team.
 }
 
 \begin{apidefinition}
 
 \begin{Csynopsis}
-int @\FuncDecl{shmemx\_team\_destroy}@(shmem_team_t newteam);
+int @\FuncDecl{shmem\_team\_destroy}@(shmem_team_t team);
 \end{Csynopsis}
 
 \begin{apiarguments}
-\apiargument{IN}{newteam}{A valid SHMEM team handle.}
+\apiargument{IN}{team}{A valid \openshmem team handle.}
 \end{apiarguments}
 
 \apidescription{
-The shmemx\_team\_destroy function destroys an existing team. This is a
+The \FUNC{shmem\_team\_destroy} function destroys an existing team. This is a
 collective call, in which every member of the team being destroyed needs
 to participate. This will free all internal memory structures associated
 with the team and invalidate the team handle. Upon return, the team
-handle is set to SHMEM\_TEAM\_NULL, after which it can no longer be
-used for team collective calls.
+handle is set to \LibConstRef{SHMEM\_TEAM\_NULL}, after which it can no longer be
```

Comment:
Post-merge note: The `team` argument is not a pointer to a team handle, so the user's handle itself cannot be set to `SHMEM_TEAM_NULL`. For consistency with `shmem_ctx_destroy()` and `free()`, I would suggest we remove the result that "the team handle is set to `SHMEM_TEAM_NULL`.